agent-container/agent-config.json

```diff
@@ -0,0 +1,2 @@
+{"author":"Amazon Web Services, Inc.","config":{"Cmd":["/agent"],"ArgsEscaped":true},"created":"2014-12-12T01:12:53.332832423Z","history":[{"created":"2014-12-12T01:12:53.332832423Z","author":"Amazon Web Services, Inc.","created_by":"[] + [] === \"\"","empty_layer":true}],"os":"linux","rootfs":{"type":"layers","diff_ids":["sha256:~~digest~~"]}}
```

Comment:
why are these static files? shouldn't they be generated at build-time?

Reply:
The diff_ids field is being generated at build-time.  Otherwise I agree maybe we can update the history.created with an actual timestamp.
These are generated by Docker otherwise -- if we don't have Docker, we need to create/provide these config files manually.  

Bottlerocket uses static configs to build its pause container: https://github.com/bottlerocket-os/bottlerocket/blob/develop/packages/ecs-agent/pause-config.json

Reply:
So do we have to ensure these files get rebuilt/committed each time then from current agent changes? Might need some sort of check to reinforce that

Reply:
Updated to string replace and use the date util to match expected formatting: `sed -i "s/~~timestamp~~/$(date +"%FT%T.%NZ")/g" image/config.json`
Otherwise we'll use https://github.com/aws/amazon-ecs-agent/blob/master/VERSION to fill in our version.